manifests/f/FireDaemon/CertifyOne/4.1.0/FireDaemon.CertifyOne.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: FireDaemon.CertifyOne
+PackageVersion: 4.1.0
+InstallerType: exe
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /qn
+  SilentWithProgress: /qb
+UpgradeBehavior: install
+ProductCode: '{0CB68A65-4E32-4C61-A8DA-A1DAABFFE1D5}'
```

```suggestion
ProductCode: 'Certify One 4.1.0'
```
This needs to match Control Panel.

```
Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\Certify One 4.1.0
```